Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -113,13 +129,34 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
         [ loadCategories ],
     );
 
+    const defaultStatusFilterId = useMemo(
+        () => {
+            if (strategiesAreLoaded && categoriesAreLoaded) {
+                const statusFilters = possibleFilters.filter(f => f.type === FilterType.Status);
+                
+                return statusFilters.filter(sf => sf.name === ContestStatus.All)[0].id;
```

Comment:
https://github.com/SoftUni-Internal/judge-worker/blob/master/OJS.Workers.Common/ExecutionStrategiesConstants.cs#L166 make this a global variable and use it here, otherwise we will have to change the default hardcoded value in multiple places

Reply:
Done.